ols/app/models/config.py

```diff
@@ -311,11 +311,20 @@ class PostgresConfig(BaseModel):
     port: PositiveInt = constants.POSTGRES_CACHE_PORT
     dbname: str = constants.POSTGRES_CACHE_DBNAME
     user: str = constants.POSTGRES_CACHE_USER
+    password_path: Optional[str] = None
     password: Optional[str] = None
     require_ssl: bool = False
     ca_cert_path: Optional[str] = None
     max_entries: PositiveInt = constants.POSTGRES_CACHE_MAX_ENTRIES
 
+    def __init__(self, **data: Any) -> None:
```

Comment:
Is the init the right place to do that? Isn't there some coercion/post init available in pydantic to do that?

Reply:
I looked at examples and every of them do it in __init__

Reply:
basically init is called when all attributes are filled in already

Reply:
ok then